letsencrypt/tests/auth_handler_test.py

```diff
@@ -148,6 +152,66 @@ def test_perform_failure(self):
         self.assertRaises(
             errors.AuthorizationError, self.handler.get_authorizations, ["0"])
 
+    @mock.patch("acme.challenges.HTTP01Response.simple_verify")
+    @mock.patch("acme.challenges.TLSSNI01Response.simple_verify")
+    @mock.patch("letsencrypt.auth_handler.logger")
+    def test_simple_verify(self, mock_logger, mock_tlssni01_verify, mock_http01_verify):
```

Comment:
It would indeed be easier that way. This would also mean you're going to mock internal methods (starting with _), and that normally impairs flexibility and refactorability, but is probably okay.